TestFoundation/TestNSString.swift

```diff
@@ -1103,37 +1103,37 @@ class TestNSString : XCTestCase {
         let trivial = NSString(string: "swift.org")
         XCTAssertEqual(trivial.substring(with: NSMakeRange(0, 5)), "swift")
 
-        let surrogatePairSuffix = NSString(string: "Hurray🎉")
-        XCTAssertEqual(surrogatePairSuffix.substring(with: NSMakeRange(0, 7)), "Hurray�")
-
-        let surrogatePairPrefix = NSString(string: "🐱Cat")
-        XCTAssertEqual(surrogatePairPrefix.substring(with: NSMakeRange(1, 4)), "�Cat")
-
-        let singleChar = NSString(string: "😹")
-        XCTAssertEqual(singleChar.substring(with: NSMakeRange(0,1)), "�")
+//        let surrogatePairSuffix = NSString(string: "Hurray🎉")
```

Comment:
Were these test failures failing, or no longer relevant? If they're broken, do we have a bug to fix them again?